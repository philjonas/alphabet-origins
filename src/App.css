.page-title{
    margin: auto;
}

.App{
    display: flex;
    flex-direction: column;
}