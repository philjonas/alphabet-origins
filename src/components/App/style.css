.page-title {
  margin: auto;
}

.App {
  display: flex;
  flex-direction: column;
}

.etymology-info{
  font-style: italic;
  font-size: 10px;
}

.link-container{
  margin: auto;
}

.link-container a{
  margin-right: 20px;
  font-style: italic;
}

@media screen and (max-width: 480px) {
  .page-title {
    margin: auto;
    font-size: 20px;
  }
}
