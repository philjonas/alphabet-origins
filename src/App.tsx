import React from 'react';
import logo from './logo.svg';
import './App.css';
import './languages/languages.css';

import Etymology from './components/Etymology/index'

const App = () => {
  return (
    <div className="App">
      <Etymology />
    </div>
  );
}

export default App;
