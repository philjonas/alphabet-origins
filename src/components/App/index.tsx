import React from 'react';
import {
  Switch,
  Route,
  Link,
} from "react-router-dom";
import '../../languages/languages.css';
import { EtymologyPage } from '../EtymologyPage'
import { WriteHieroglyphics } from '../WriteHieroglyphics'
import './style.css';

export const App = () => {
  return (
    <div className="App">
      <h1 className="page-title">The Hieroglyphic Origins of the Greek and Latin Alphabets</h1>
      <div className="link-container">
        <Link to="/">Home</Link>
        <Link to="/write">Write in Hieroglyphics and Phoenician</Link>
      </div>
      <Switch>
        <Route exact path="/write">
          <WriteHieroglyphics />
        </Route>
        <Route exact path="/">
          <EtymologyPage />
        </Route>
        <Route exact path="/hieroglyphics/:id">
          <EtymologyPage />
        </Route>
        <Route path="*">
          <EtymologyPage />
        </Route>
      </Switch>
      <p className="etymology-info">Information taken from <a href="https://en.wikipedia.org/wiki/Phoenician_alphabet" target="_blank" rel="noreferrer"> Wikipedia's Phoenician Alphabet</a> page</p>
    </div>
  );
}
