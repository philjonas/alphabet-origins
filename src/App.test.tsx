import React from 'react';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })
import { App } from './App';


test('matches snapshot', () => {
  const wrapper = shallow(<App />)
  expect(toJson(wrapper)).toMatchSnapshot();
})