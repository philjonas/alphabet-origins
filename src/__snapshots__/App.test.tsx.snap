// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <h1
      className="page-title"
    >
      The Hieroglyphic Origins of the Greek and Latin Alphabets
    </h1>
    <Connect(HieroglyphicsTemplate) />
    <Switch>
      <Route
        path="/:id"
      >
        <Connect(EtymologyPageTemplate) />
      </Route>
      <Route
        path="*"
      >
        <Connect(EtymologyPageTemplate) />
      </Route>
    </Switch>
    <p
      className="etymology-info"
    >
      Information taken from 
      <a
        href="https://en.wikipedia.org/wiki/Phoenician_alphabet"
        target="_blank"
      >
         Wikipedia's Phoenician Alphabet
      </a>
       page
    </p>
  </BrowserRouter>
</div>
`;
